components/automate-ui/src/app/entities/service-groups/service-groups.actions.ts

```diff
@@ -21,7 +21,11 @@ export enum ServiceGroupsActionTypes {
 
   GET_SERVICES_BY_SERVICE_GROUP = 'SERVICE_GROUPS::SERVICES::GET',
   GET_SERVICES_BY_SERVICE_GROUP_SUCCESS = 'SERVICE_GROUPS::SERVICES::GET::SUCCESS',
-  GET_SERVICES_BY_SERVICE_GROUP_FAILURE = 'SERVICE_GROUPS::SERVICES::GET::FAILURE'
+  GET_SERVICES_BY_SERVICE_GROUP_FAILURE = 'SERVICE_GROUPS::SERVICES::GET::FAILURE',
+
+  GET_SERVICE_GROUPS_SUGGESTIONS = 'GET_SERVICE_GROUPS_SUGGESTIONS::SERVICES::GET',
```

Comment:
To follow the name standard above I would suggest calling it like this:
```suggestion
  GET_SERVICE_GROUPS_SUGGESTIONS = 'SERVICE_GROUPS::SUGGESTIONS::GET',
```
Same for the two below 👇

Reply:
@afiune updated 👍🏻